### ISSUE TITLE
Deleted IMAP Mail shows twice in Trash Folder

### ISSUE THREAD

**Issue by Naklarsuper** [k9mail/k-9#3939]
I am using Yahoo Mail on K9 via IMAP. When I delete an email on K9 it immediately moves it to Trash. Then it seems to replicate the folder to Yahoo's Trash, where it also appears (e.g., on PC). THEN it replicates again to K9, where the deleted Email shows up in the Trash folder twice. 

Emptying Trash on Yahoo Mail (on PC) removes the deleted mail both on Yahoo as also in K9, but leaves one in place, that can only be removed by "empty trash" in K9

### Expected behavior
would like to see a deleted email only once, not twice in K9

### Actual behavior
deleted mail shows in trash, then after updating folders shows a second time. Each mail deleted on K9 is there twice

### Steps to reproduce
1.
2.
3.

### Environment
K-9 Mail version:

Android version: Pie

Account type (IMAP, POP3, WebDAV/Exchange): IMAP, Yahoo

Please take some time to [retrieve logs](https://github.com/k9mail/k-9/wiki/LoggingErrors) and attach them here:

**Comment by cketti** [comment]
Duplicate of #2705
